content/en/docs/setup/cri.md

```diff
@@ -0,0 +1,169 @@
+---
+reviewers:
+- vincepri
+- bart0sh
+title: CRI installation
+content_template: templates/concept
+weight: 100
+---
+{{% capture overview %}}
+Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+This page contains installation instruction for various runtimes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+### Docker
+
+On each of your machines, install Docker.
+Version 17.03 is recommended, but 1.11, 1.12 and 1.13 are known to work as well.
+Versions 17.06+ _might work_, but have not yet been tested and verified by the Kubernetes node team.
```

Comment:
`Versions 18.06+ _might work_, but have not yet been fully verified.`

AFAIK, sig node are not going to verify any newer docker versions.
which leaves us verifying newer docker only via e2e tests.